loader/loader.c

```diff
@@ -2230,14 +2269,19 @@ static VkResult loader_read_layer_json(const struct loader_instance *inst, struc
 
     strncpy(props->info.layerName, name, sizeof(props->info.layerName));
     props->info.layerName[sizeof(props->info.layerName) - 1] = '\0';
+    if (0 != strncmp(props->info.layerName, "VK_LAYER_", 9)) {
+        loader_log(inst, VULKAN_LOADER_ERROR_BIT, 0, "Layer name %s does not conform to naming standard (Policy #LLP_LAYER_3)",
```

Comment:
Is this an ERROR or a Warning? I ask cause the policy statement says that new layers should conform to the naming convention, not that all layers should. This would cause needless false positives on peoples systems.

Reply:
I'll change this to a warning for now.